stock_picking_location_info/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019  Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Stock picking location info',
```

Comment:
IMHO the module name is too generic according to the field introduced (location_putaway_dest_id).

Either the aim is to make a generic module (base one) to add further information on picking level, either the module has to be renamed